samples/config/prod/deploy.sh

```diff
@@ -9,10 +9,10 @@
 
 #### Variables
 
-NAMESPACE="bench"
+NAMESPACE="prod"
 DOMAIN="frk8s.net"
 URL_PREFIX="openam"
-TYPE="m-cluster"
+TYPE="s-cluster"
```

Comment:
We are going to get these from an env.sh file, right? 

At a miniumum we should allow overrides
DOMAIN=${DOMAIN:example.com}